net-im/err/err-6.1.8.ebuild

```diff
@@ -52,6 +52,7 @@ distutils_enable_tests pytest
 
 python_prepare_all() {
 	sed -i -e '/pygments-markdown-lexer/d' setup.py || die
+	sed -i -e '378d' errbot/backends/irc.py || die
```

Comment:
This isn't very reliable or able to live between releases. Could you match some exact pattern, or preferably patch this out?

Reply:
It's a fix for this specific version, it won't be needed in next release. So should I make a patch too?  Because in next release, neither this sed nor patch will need.

Reply:
Yeah I find it better that way. I don't trust line nr based removals. Besides I'm sure we'll see a copy-paste for the next version bump... happens every time. 

Patches are supposed to be temporary - they're supposed to be upstreamed. Removing patches with version bumps means something correct has been done. Also, you could package a snapshot version? In other words, do a version bump?

Reply:
Done :)